WebDriverAgentTests/IntegrationTests/FBScreenTests.m

```diff
@@ -31,7 +31,7 @@ - (void)testScreenScale
 - (void)testStatusBarSize
 {
   CGSize statusBarSize = [FBScreen statusBarSizeForApplication:self.testedApplication];
-  XCTAssertFalse(CGSizeEqualToSize(CGSizeZero, statusBarSize));
+  XCTAssertTrue(CGSizeEqualToSize(CGSizeZero, statusBarSize));
```

Comment:
Maybe, zero is not strange. `statusBars.firstMatch` instead of `statusBars.fb_firstMatch` was also zero.

https://stackoverflow.com/questions/57590252/ios-13-status-bar-hidden-for-all-iphone-types-in-landscape
https://stackoverflow.com/questions/57608752/ios-13-status-bar-style